projects/js-packages/components/components/social-service-icon/style.scss

```diff
@@ -0,0 +1,28 @@
+// TODO: @mkm119 replace these with gutenber-base-styles once that file is moved to js-packages/base-styles
```

Comment:
I wonder if we should wait for that and update it in this PR. If we can get that file in `base-styles` over the next couple of days, it won't hold me up. I'm not sure if there's anything else that is waiting on this icon component.

Reply:
I can update this as part of the next PR if this gets merged before that one @pablinos